### ISSUE TITLE
OT support any unlocked book for tW tool

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#5204]
### User Story

As a user, I want to check my OT book in the tW tool so that I can finish checking the OT.

### Features / Specifications

- [ ] If there is:
  - [ ] Aligned OT book
  - [ ] tW articles
- [ ] then: make that book available for checking in that GL in the tW tool

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by PhotoNomad0** [comment]
@benjore - does there need to be a minimum checking level to use a book as a GL in tW?

**Comment by PhotoNomad0** [comment]
Didn't it use to be checking level of at least 3?  That's what it seems to be now.

**Comment by benjore** [comment]
yes. That is correct. Just found it... https://gl-manual.readthedocs.io/en/latest/gl_translation.html#what-needs-to-be-translated

**Comment by PhotoNomad0** [comment]
@benjore - to clarify: for TW articles, does this mean that both:
- must have tw articles in the GL
- must have tw groups in the OL for the book (or to put it another way - the book has tw tags in it)

**Comment by benjore** [comment]
If OL means Original Language, then yes.

This is what is needed: 
- [ ] In the Original Language
  - [ ] tW words tagged in the Hebrew/Greek (per book)
- [ ] In the Gateway Language
  - [ ] translated tW articles published to Door43 Catalog, Level 2 
  - [ ] book of the Bible aligned and published in Door43 Catalog, Level 3 (per book)

**Comment by PhotoNomad0** [comment]
@cckozie - Have a PR for this issue, but note that we currently have no OT aligned books in Hebrew.  So I added hbo.zip to https://github.com/unfoldingWord-dev/translationCore/pull/5763 with alignment to Ruth 1:1 so that you can test this.

**Comment by PhotoNomad0** [comment]
@cckozie - verified in https://tc-builds.door43.org/build/develop/translationCore-macos-x64-1.2.0-ee07c41.dmg

See #5763 for the replacement for `resources/⁨en⁩/⁨bibles⁩/⁨ult⁩/⁨v1⁩/rut` that will allow you to open Ruth in tW.  I did an alignment of Ruth 1:1 so we could test.  Waiting on real aligned OT ult books before more can be done.

**Comment by cckozie** [comment]
1.2.0 (c5c12e1)
With the aligned Ruth ULT resource I am able to select either English or Hindi as the gl on the tW tool card. I should only be able to select English since there is not yet an aligned Hindi.

The following was moved to issue #5813.

~~If I make an edit in tW that invalidates the selection, it words as expected.
However, if after making a selection in tW I then make an edit in wA that invalidates the selection, the tW tool card then indicates 1 broken link, but it still indicates 100% complete, even after loading the project in tW and returning to the tools page.
This does not seem to be the case in NT books. I made selections for all tW Name checks in a Titus project. I made and edit in wA that reset one of the selections. The tW tools card was updated correctly.~~
![image](https://user-images.githubusercontent.com/8171759/52807948-7337d080-3052-11e9-855b-3122ef728e28.png)
